content/en/docs/concepts/overview/working-with-objects/finalizers.md

```diff
@@ -22,7 +22,7 @@ and does the following:
 
   * Modifies the object to add a `metadata.deletionTimestamp` field with the
     time you started the deletion.
-  * Prevents the object from being removed until its `metadata.finalizers` field is empty.
+  * Prevents the object from being removed until its `metadata.finalizers` field is removed as `finalizers` can not be added or changed.
```

Comment:
```suggestion
  * Prevents the object from being removed until its `metadata.finalizers` field empty.
    (Once deletion begins for an object, the only modification permitted to `metadata.finalizers` is to
    remove one or more existing finalizers.)
```

Reply:
this change is required to make the sentence work

Reply:
Hi @sftim , im a bit confused with this statement actually as whenever we need to remove object we usually patch remove the finalizer, then why are we mentioning it as `empty`?
Or is it because we want to mention the `metadata.finalizer` field?

Reply:
I'll say we write ,
`the finalizer keys or the finalizers are removed 
`Or
`the finalizer list is empty,`
because only saying finalizer is empty seems confusing to its concept.

Reply:
Maybe

```markdown
… `metadata.finalizers` field has become empty: a list with no members.
```

Reply:
It's hard to find out which behavior is true because as soon as the list becomes empty, Kubernetes deletes the object. Maybe SIG API Machinery can recommend what to explain here?

Reply:
"as soon as all items are removed from the finalizers list" is probably sufficient. Both an empty list and a completely absent finalizers field have the same effect